cmd/bmp180/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// bmp180 reads the current temperature and pressure from a BMP180.
```

Comment:
Sure!

BTW sorry for the feeling of nitpicking fest; I'm trying to keep a very specific style on the library and for the device drivers I really want to behave the best w.r.t. the device (e.g. like sleeping the exact time as specified by the datasheet). I hope it's not a bummer for you.